app/scripts/services/validateRegistrant.js

```diff
@@ -233,7 +235,6 @@ angular
       var blocks = page
         ? _.find(conference.registrationPages, { id: page }).blocks
         : _.flatten(_.map(conference.registrationPages, 'blocks'));
-
       var that = this;
       angular.forEach(blocks, function(block) {
```

Comment:
If you make `function(block) { ... }` an arrow function you don't have to do the `that = this` hack.

I know you didn't write this code... You can leave it...